cli/commands/run/flags.go

```diff
@@ -0,0 +1,437 @@
+			Name:        IAMAssumeRoleWebIdentityTokenFlagName,
+			EnvVars:     flags.EnvVars(IAMAssumeRoleWebIdentityTokenFlagName),
+			Destination: &opts.IAMRoleOptions.WebIdentityToken,
+			Usage:       "For AssumeRoleWithWebIdentity, the WebIdentity token.",
+		}, TerragruntIAMWebIdentityTokenFlagName),
+
+		flags.BoolWithDeprecatedFlag(opts, &cli.BoolFlag{
+			Name:        QueueIgnoreErrorsFlagName,
+			EnvVars:     flags.EnvVars(QueueIgnoreErrorsFlagName),
+			Destination: &opts.IgnoreDependencyErrors,
+			Usage:       "Continue processing Units even if a dependency fails.",
+		}, TerragruntIgnoreDependencyErrorsFlagName),
+		flags.BoolWithDeprecatedFlag(opts, &cli.BoolFlag{
+			Name:        QueueIgnoreDAGOrderFlagName,
+			EnvVars:     flags.EnvVars(QueueIgnoreDAGOrderFlagName),
+			Destination: &opts.IgnoreDependencyOrder,
+			Usage:       "Ignore DAG order for --all commands.",
+		}, TerragruntIgnoreDependencyOrderFlagName),
+		flags.BoolWithDeprecatedFlag(opts, &cli.BoolFlag{
+			Name:        QueueExcludeExternalFlagName,
+			EnvVars:     flags.EnvVars(QueueExcludeExternalFlagName),
+			Destination: &opts.IgnoreExternalDependencies,
+			Usage:       "Ignore external dependencies for --all commands.",
+		}, TerragruntIgnoreExternalDependenciesFlagName),
+		flags.BoolWithDeprecatedFlag(opts, &cli.BoolFlag{
+			Name:        QueueIncludeExternalFlagName,
+			EnvVars:     flags.EnvVars(QueueIncludeExternalFlagName),
+			Destination: &opts.IncludeExternalDependencies,
+			Usage:       "Include external dependencies for --all commands without asking.",
+		}, TerragruntIncludeExternalDependenciesFlagName),
+		flags.GenericWithDeprecatedFlag(opts, &cli.GenericFlag[int]{
+			Name:        ParallelismFlagName,
+			EnvVars:     flags.EnvVars(ParallelismFlagName),
+			Destination: &opts.Parallelism,
+			Usage:       "Parallelism for --all commands.",
+		}),
+		flags.GenericWithDeprecatedFlag(opts, &cli.GenericFlag[string]{
+			Name:        QueueExcludesFileFlagName,
+			EnvVars:     flags.EnvVars(QueueExcludesFileFlagName),
+			Destination: &opts.ExcludesFile,
+			Usage:       "Path to a file with a list of directories that need to be excluded when running *-all commands.",
+		}, TerragruntExcludesFileFlagName),
+		flags.SliceWithDeprecatedFlag(opts, &cli.SliceFlag[string]{
+			Name:        QueueExcludeDirFlagName,
+			EnvVars:     flags.EnvVars(QueueExcludeDirFlagName),
+			Destination: &opts.ExcludeDirs,
+			Usage:       "Unix-style glob of directories to exclude from the queue of Units to run.",
+		}, TerragruntExcludeDirFlagName),
+		flags.SliceWithDeprecatedFlag(opts, &cli.SliceFlag[string]{
+			Name:        QueueIncludeDirFlagName,
+			EnvVars:     flags.EnvVars(QueueIncludeDirFlagName),
+			Destination: &opts.IncludeDirs,
+			Usage:       "Unix-style glob of directories to include from the queue of Units to run.",
+		}, TerragruntIncludeDirFlagName),
+		flags.BoolWithDeprecatedFlag(opts, &cli.BoolFlag{
+			Name:        DebugInputsFlagName,
```

Comment:
I'm sorry to do this to you, @levkohimins , but can we change this to `inputs-debug` while we're making this change.

We've been discussing the need to add other flags like:
- `inputs-disable`
- `inputs-allow-list`

and I think it's best that we group these under the same `inputs` prefix.

Reply:
No problem at all. Done.